library/Jira/RestApi.php

```diff
@@ -125,22 +127,24 @@ public function updateIssue(IssueUpdate $update)
 
     protected function prepareIssueQuery($host = null, $service = null, $onlyOpen = true)
     {
+        $config = Config::module('jira');
```

Comment:
Fetching the setting and setting the default in multiple places feels redundant, you can address this if you want - but I do not see it as a requirement to get this merged.